.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: dart
+
+dart:
+  - dev
+
+dart_task:
+  - test: -p vm
```

Comment:
There are tests that are specifically for the browser, can I also just add
test -p chrome
here?